src/main/java/org/dwcj/controls/AbstractCompoundControl.java

```diff
@@ -1,7 +1,7 @@
 package org.dwcj.controls;
```

Comment:
This one should be removed. Div is the control that should be used as a building block